deeptools/_version.py

```diff
@@ -2,4 +2,4 @@
 # This file is originally generated from Git information by running 'setup.py
 # version'. Distribution tarballs contain a pre-generated copy of this file.
 
-__version__ = '1.6.0'
+__version__ = '1.6.0-173-gc811461'
```

Comment:
I just did that. Still not sure what will happen when I want to commit a
change on that file

On Thu, Dec 3, 2015 at 9:21 AM, Devon Ryan <notifications@github.com> wrote:

> In deeptools/_version.py
> <https://github.com/fidelram/deepTools/pull/118#discussion_r46523567>:
>
> > @@ -2,4 +2,4 @@
> >  # This file is originally generated from Git information by running 'setup.py
> >  # version'. Distribution tarballs contain a pre-generated copy of this file.
> >
> > -__version__ = '1.6.0'
> > +__version__ = '1.6.0-173-gc811461'
>
> Maybe this should be in .gitignore then.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fidelram/deepTools/pull/118/files#r46523567>.
>



-- 

Fidel Ramirez